stripe/src/main/java/com/stripe/android/model/StripeIntent.kt

```diff
@@ -220,5 +225,15 @@ interface StripeIntent : StripeModel {
                 ) : Parcelable
             }
         }
+
+        @Parcelize
+        object BlikAuthorize : NextActionData() {
+            override fun hashCode(): Int {
+                return 0
+            }
+            override fun equals(other: Any?): Boolean {
+                return true
+            }
```

Comment:
remove

Reply:
seems not compilable without explicitly override this as the object instance without any parameterized field won't be able to tell how to parcel itself, also tried replacing with a regular `data class` and get similar compilation error

Reply:
`@Parcelize` can apply to an `object`

Reply:
We can examine this in a future PR